bot/exts/info/tags.py

```diff
@@ -24,145 +29,214 @@
 FOOTER_TEXT = f"To show a tag, type {constants.Bot.prefix}tags <tagname>."
 
 
+class COOLDOWN(enum.Enum):
+    """Sentinel value to signal that a tag is on cooldown."""
+
+    obj = object()
```

Comment:
What's the difference to this and `COOLDOWN = object()`?

I assume this has the benefit of `__repr__` right?

Reply:
I used an Enum because I couldn't find a way to type the sentinel alone